src/query/fuzzy_query.rs

```diff
@@ -34,22 +33,6 @@ impl Automaton for DfaWrapper {
     }
 }
 
-/// A range of Levenshtein distances that we will build DFAs for our terms
-/// The computation is exponential, so best keep it to low single digits
-const VALID_LEVENSHTEIN_DISTANCE_RANGE: Range<u8> = 0..3;
```

Comment:
I don't want to let user build arbitrary distance. Above 5, computer catch fires.
We still need a guard and return an error. I think <3 is a reasonable limit.